ui/src/components/LineLoader/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Is this component in use? if not, please remove

Reply:
Thanks for the feedback.  This component is used in a module that is not part of this current pr.  However, this pr encompasses core functionality and components that can be used by modules.  Hence, the reason why I've included it here.  Along with the `DownLoadJsonButton` component.